src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -252,6 +250,15 @@ private static boolean shouldIndent(String currentNodeName) {
                  || "body".equals(currentNodeName));
     }
 
+    private static boolean isTinymceStyleDiv(Node attribute) {
+        if (!"style".equalsIgnoreCase(attribute.getNodeName())) {
+            return false;
```

Comment:
Does this not always return false since this is called from a `div` node not a `style` node?

Reply:
This is checking the `div`'s attributes, i.e. `<div style="..."`.

Context for why this is necessary: the newer version of tinymce adds this div for every tinymce editor being instantiated:
```html
<div style="position: static; height: 0px; width: 0px; padding: 0px; margin: 0px; border: 0px;"></div>
```
and in Chrome/Windows/... (not sure which one but in AppVeyor setting) it shows as:
```html
<div style="position: static; height: 0px; width: 0px; padding: 0px; margin: 0px; border: 0px none;"></div>
```